### ISSUE TITLE
Coredump in `msLayoutTextSymbol` when labeling "null"

### ISSUE THREAD

**Issue by mkofahl** [mapserver/mapserver#5298]
MapServer segfaults when it tries to determine the length of an annotation text which is actually `null`.

Backtrace:
```
Program received signal SIGSEGV, Segmentation fault.
#0  strlen () at ../sysdeps/x86_64/strlen.S:106
#1  0x00007ffff7aad35d in msLayoutTextSymbol (map=<optimized out>, ts=0x7fffffffdf20, tgret=0x62d8e0)
#2  0x00007ffff7a1b768 in msDrawLabel (map=0x62a6b0, image=0x65a910, labelPnt=..., string=0x0, label=0x6534e0, scalefactor=1.3333333333333333)
#3  0x00007ffff7a1bf72 in pointLayerDrawShape (map=0x62a6b0, image=0x65a910, layer=0x651c50, shape=0x7fffffffe220, drawmode=<optimized out>)
```

I see an assert in `msLayoutTextSymbol` https://github.com/mapserver/mapserver/blob/branch-7-0/textlayout.c#L446 and wonder why it doesn't fails. gdb tells me `annotext` to be `0x0`, both in L446 and in https://github.com/mapserver/mapserver/blob/branch-7-0/textlayout.c#L486 where the segfault happens.

**Comment by tbonfort** [comment]
Martin,
can you provide a backtrace from a debug build please?

**Comment by mkofahl** [comment]
Worked on an older branch again. Sorry for the noice. Although i couldn't find the real reason yet, the segfault is lready fixed with #5240 or #5241.
